crates/parser2/src/ast/stmt.rs

```diff
@@ -63,12 +63,7 @@ ast_node! {
     SK::AssignStmt,
 }
 impl AssignStmt {
-    /// Returns the pattern of the lhs of the assignment.
-    pub fn pat(&self) -> Option<super::Pat> {
-        support::child(self.syntax())
-    }
-
-    /// Returns the expression of the rhs of the assignment.
+    /// Returns the expression of the lhs and rhs of the assignment.
     pub fn expr(&self) -> Option<super::Expr> {
         support::child(self.syntax())
     }
```

Comment:
This is incorrect; please refer to `ast::expr::BinExpr` implementation as an example.

Reply:
Just fixed it, I'll start working on `AugAssignStmtScope`, and then will squash the commits.